Automattic/jetpack
# Social: Connections API schema front end changes

Related to #40677

## Proposed changes:

* Use the updated connections schema on the front-end following #40607

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to connections management
* Verify that the connection name is displayed correctly for all the connections
* Remove the display name from your Bluesky profile and reconnect it
* Verify that the display name for the connection is the blueksy handle
* Add a Mastodon connection and then break the connection by removing Jetpack [here](https://mastodon.social/oauth/authorized_applications).
* Change the tab in the edior and come back or reload the page in case of settings page to have the broken connection reflected
* In the editor, verify that the broken connection notice shows up fine.
* Reconnect the broken Mastodon account
* Verify that reconnection works fine (with the correct handle)
* In editor, toggle the connections ON/OFF to confirm it works fine
* Try to send post only to 1 network, rest toggled off to verify it works
* If you have an unsupported network like Twitter added, verify it's marked as unsupported. I verified it for my old Twitter connection.
* In the Jetpack sidebar, click on Preview social posts
* Verify that the connections are displayed fine
